ui/pages/home/index.scss

```diff
@@ -136,24 +136,35 @@
 
   &__new-network-notification {
     margin-bottom: 50px;
-  }
-
-  &__new-network-notification-message {
-    display: flex;
-    flex-direction: row;
 
     &--image {
       margin-right: 10px;
     }
-  }
 
-  &__close {
-    color: var(--ui-black);
-    background: none;
-    margin-left: 20px;
+    &--close {
+      color: var(--ui-black);
+      background: none;
+      margin-left: 20px;
+    }
   }
 
   &__error-message {
     left: 8px;
   }
+
+  &__new-nft-notification {
+    margin-bottom: 24px;
+    margin-right: 8px;
+    left: 16px;
+
+    &--image {
```

Comment:
Remember that `--` in BEM is used to designate a variant of the element in question. If you're adding an element inside of another element then you probably only want one dash here (the full class names in that case would be `.home__new-nft-notification-image` and `.home__new-nft-notification-close` instead of `.home__new-nft-notification--image` and `.home__new-nft-notification--close`).